src/components/Checkbox/Checkbox.js

```diff
@@ -49,14 +49,13 @@ Checkbox.propTypes = {
   className: PropTypes.string,
   disabled: PropTypes.bool,
   id: PropTypes.string.isRequired,
-  labelText: PropTypes.node,
+  labelText: PropTypes.node.isRequired,
```

Comment:
Jumping in - the reason why this was a problem was because people were making composed UI out of the elements. Requiring the label hurt the 'pro' user who was handling the a11y differently

Reply:
@chrisdhanaraj hi chris! 👋 

Do you have a code snippet in mind? Am down to implement a custom `prop` checker logic to accommodate what you are describing 😄

Reply:
@chrisdhanaraj any input here? 😄

Reply:
Oh sorry, ignore me! For some reason I was mixing up the Checkbox heading and the checkbox label :) my total bad

Reply:
Lol @chrisdhanaraj way to keep us in limbo here!

Reply:
https://media.giphy.com/media/d2YZg3m63zNgWa6k/giphy.gif

Reply:
Wow that's a high resolution gif right there